Lib/test/test_capi/test_opt.py

```diff
@@ -141,89 +56,8 @@ def get_opnames(ex):
 
 @requires_specialization
 @unittest.skipIf(Py_GIL_DISABLED, "optimizer not yet supported in free-threaded builds")
-@unittest.skipUnless(hasattr(_testinternalcapi, "get_optimizer"),
```

Comment:
I think we want to keep this test. It tests important things about the internal executor API.